src/models/wc.js

```diff
@@ -0,0 +1,20 @@
+import { round } from "../utilities/utilities.js";
+
+/**
+ * Calculates the Wind Chill Index (WCI) in accordance with the ASHRAE 2017 Handbook Fundamentals - Chapter 9 [18]_.
+ * @param {number} tdb - dry bulb air temperature,[°C]
+ * @param {number} v - wind speed 10m above ground level, [m/s]
+ * @param {Object} [kwargs] (Optional) Other parameters.
+ * @param {boolean} [kwargs.round=true] - If True rounds output value, if False it does not round it.
+ * @returns {object} {"wci": wci} - wind chill index, [W/m2)]
```

Comment:
Does this work? Do you get intellisense on the result when you use the function?